app/models/session.py

```diff
@@ -46,7 +47,7 @@ class Session(SoftDeletionModel):
     event_id = db.Column(db.Integer, db.ForeignKey('events.id', ondelete='CASCADE'))
     creator_id = db.Column(db.Integer, db.ForeignKey('users.id', ondelete='CASCADE'))
     state = db.Column(db.String, default="pending")
-    created_at = db.Column(db.DateTime(timezone=True), default=datetime.datetime.utcnow)
+    created_at = db.Column(db.DateTime(timezone=True), default=dateandtime.now())
```

Comment:
Why different from func.now()?

Reply:
Because func imported from sqlAlchemy is been used in the file already. And  I had to import func from sqlAlchemy.sql, but I imported it using `as`.